plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -168,18 +157,23 @@ public function test_img_sizes_is_equal_to_picture_source_sizes_for_picture_elem
 
 		$img_markup = apply_filters( 'the_content', $image );
 
-		// Apply picture element support.
-		$this->opt_in_to_picture_element();
-
-		$picture_markup = apply_filters( 'the_content', $image );
-
 		$img_processor = new WP_HTML_Tag_Processor( $img_markup );
 		$this->assertTrue( $img_processor->next_tag( array( 'tag_name' => 'IMG' ) ), 'There should be an IMG tag.' );
 		$img_sizes = $img_processor->get_attribute( 'sizes' );
+		$img_src   = $img_processor->get_attribute( 'src' );
 
+		$this->assertStringEndsWith( '.jpg', $img_src, 'Should return .jpg img src.' );
```

Comment:
Can you explain why the image `src` would be expected to be a JPEG here instead of WebP and why that is relevant to the this test?

Reply:
Thanks for raising Joe. I understood that i mixup the test and it should be Webp instead of JPEG. apologize for that.

I missed because when test uploaded image in test should return webp instead of jpeg image in our test it return jpeg image 🤔 

```
// Create an image.
$attachment_id = self::factory()->attachment->create_upload_object( TESTS_PLUGIN_DIR . '/tests/data/images/leaves.jpg' );

// Create some content with the image.
$image = wp_get_attachment_image(
	$attachment_id,
	'large',
	false,
	array(
		'class' => "wp-image-{$attachment_id}",
		'alt'   => 'Green Leaves',
	)
);
```

Reply:
Resolved now with https://github.com/WordPress/performance/pull/1408/commits/0e07201f88482e95f979cfe7eb5884245ecb69cc